src/Files.App/Views/PaneHolderPage.xaml.cs

```diff
@@ -74,13 +76,9 @@ private bool windowIsCompact
 
 		public bool IsMultiPaneActive => IsRightPaneVisible;
 
-		public bool IsMultiPaneEnabled
-		{
-			get => !(App.Window.Bounds.Width <= 750);
-		}
+		public bool IsMultiPaneEnabled => !(App.Window.Bounds.Width <= 750);
```

Comment:
Can you add back the brackets?

Reply:
Yeah, can you show the guild-line for this?

Reply:
set is not defined. so I think we can leave as it is.

Reply:
I find having it on a separate line is more readable

Reply:
Hoe about being like this?

```cs
public bool IsMultiPaneEnabled
     => !(App.Window.Bounds.Width <= 750);
```

Reply:
That's certainly better

Reply:
Fixed